RELEASE-NOTES.md

```diff
@@ -183,7 +187,6 @@ Changes in the `Entity` hierarchy:
 * Changed the constructor signature of `Property`
 * Removed `Entity::setClaims` (`Item::setClaims` has been retained)
 * Removed `Entity::stub`
-* Removed `Property::newEmpty`
```

Comment:
I don't get it. Are you talking about the method here in the README or in the code? Looking at it again, this change in the readme still seems wrong